core/services/vrf/vrf_simulate_blockchain_test.go

```diff
@@ -132,28 +106,8 @@ func TestIntegration_RandomnessRequest(t *testing.T) {
 	require.NoError(t, err, "seed %x out of range", seed)
 	_, err = goProof.CryptoProof(vrf.PreSeedData{
 		PreSeed:   preSeed,
-		BlockHash: requestlog.BlockHash,
-		BlockNum:  uint64(blockNum),
+		BlockHash: r.Raw.Raw.BlockHash,
+		BlockNum:  uint64(r.Raw.Raw.BlockNumber),
 	})
 	require.NoError(t, err, "problem verifying solidity proof")
-
-	// Check that a log from a different address is rejected. (The node will only
-	// ever see this situation if the ethereum.FilterQuery for this job breaks,
-	// but it's hard to test that without a full integration test.)
```

Comment:
I think this was a regression test for something we've slipped on before. Is there somehow an equivalent check in the new model?

Reply:
@se3000 this part of the test can't really happen with the simulated backend because we can't (easily) create fake contract logs to feed to the VRF service. The logs are created automatically from the contract txs. I have, however, added `randomness_log_event_test.go` which adds a test for the same validation that the fake tx was trigging in the integration test. I think that will suffice.